wfl/calculators/espresso.py

```diff
@@ -140,9 +86,9 @@ def calculate(self, atoms=None, properties=_default_properties, system_changes=a
             calculation_succeeded = True
             if 'DFT_FAILED_ESPRESSO' in atoms.info:
                 del atoms.info['DFT_FAILED_ESPRESSO']
-            if "__calculator_output_prefix" in atoms.info:
-                save_calc_results(atoms, prefix=atoms.info["__calculator_output_prefix"], properties=properties)
-                atoms.info["__calculator_results_saved"] = True
+            if "_output_prefix" in atoms.info:
```

Comment:
Please undo this change.  If the `espresso` calculator modifies the configurations and breaks the ASE caching (the way the Vasp calculator does), you do need to save the results, but you need to use the property that's set by  `calculator.generic`, which is the previous version of this bit of code, i.e. `__calculator_output_prefix`. 

I actually suspect that this calculator doesn't modify the configurations, in which case the ASE caching should work, and you don't need to save the results explicitly here at all, but maybe this PR isn't the right place to make that change.

Reply:
Undid the change in #325 . Maybe let's see in #325 the calculator modifies the configs and remove if not.